src/mumble/CoreAudio.cpp

```diff
@@ -37,7 +37,7 @@ class CoreAudioInit : public DeferInit {
 		CoreAudioInputRegistrar *cairReg;
 		CoreAudioOutputRegistrar *caorReg;
 	public:
-		CoreAudioInit() : cairReg(NULL), caorReg(NULL) {}
+		CoreAudioInit() : cairReg(nullptr), caorReg(nullptr) {}
```

Comment:
While we're at it, let's remove this constructor's implementation and set the defaults in the header file.